src/components/navigation/NavItem.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-  <li :class="{'aui-nav-selected': this.selected}">
-    <a :href="href">{{name}}</a>
+  <li class="nav-item" :class="{'aui-nav-selected': this.selected}">
```

Comment:
What's the reason to add this `nav-item` class? Is it used anywhere?

Reply:
Not sure. I think I used for something but there's no use for it anymore.